pkgs/servers/http/envoy/default.nix

```diff
@@ -140,6 +140,9 @@ buildBazelPackage rec {
     "--cxxopt=-Wno-error=type-limits"
 
     "--define=wasm=${wasmRuntime}"
+
+    # prevent builds from being oomkilled
+    "--jobs=8"
```

Comment:
This should use NIX_BUILD_CORES.

Reply:
Got it -- turns out Bazel was already using that to do the same thing -- I removed this line completely.